charybdis-macros/src/traits/fields/query.rs

```diff
@@ -22,7 +22,7 @@ impl FieldsQuery for Vec<&Field<'_>> {
     fn insert_bind_markers(&self) -> String {
         let str_vec = self
             .iter()
-            .map(|field| format!(r#":"{}""#, field.name))
```

Comment:
@dracarys18 I don't think we need double quotes within insert bind markers, right?

Reply:
We do need it

Reply:
Does db raise an error even for bind markers? Then we might need it for other clauses. I will work on test suite on charbidis by the end of this year so we can avoid this iterations.

Reply:
Ah ok. Then we need to test all clauses, update migrations queries as well etc. I will create issue so we adress all of these, and we need to write test cases.

Reply:
Or we don't allow reserved keywords as column names at all?

Reply:
I don't think that's the right solution, Let's say for me I am moving my application from Postgres to Cassandra. And one of the columns named `token` happened to be a keyword in Cassandra, Now I don't want to change the column name just because it's a keyword in cassandra.

I think adding escape characters to every column is the best solution for all.

Reply:
Well, you could use serde rename it to 'token' for deseriazlier if it's needed for data exchange. However, I'll create issue and if we want to support it, we need to update all queries within the lib.

Reply:
I mean the column name would be picked up by the macro from the struct and stringified, So if I rename the struct field queries would break since the column name is not changed

Reply:
That is my point, you use something else for db e.g. 'token_field' and serde to rename it to "token" for data exchange. Otherwise, I don't see why it is the problem if you are rewriting the code anyway - for Cassandra in this case.